compiler/ml/parsetree.ml

```diff
@@ -372,9 +372,7 @@ and jsx_prop =
       Location.t
       * expression
 
-and jsx_children =
-  | JSXChildrenSpreading of expression
-  | JSXChildrenItems of expression list
+and jsx_children = JSXChildrenItems of expression list
```

Comment:
This can be removed now as well, right? Should probably do that in the same PR.

Reply:
fixed!